### ISSUE TITLE
katacoda: Stages scenario seems to never start

### ISSUE THREAD

**Issue by iesahin** [iterative/dvc.org#2184]
https://katacoda.com/dvc/courses/get-started/stages scenario has 

```sh
~$ :;: ===================================
~$ :;: Please wait until DVC is installed.
~$ :;: ===================================
~$ until hash dvc &>/dev/null; do sleep 1; done
```

at start but never seems to end. It looks it's not installed after `C-c`. 

```sh
~$ until hash dvc &>/dev/null; do sleep 1; done
^C  
~$ dvc --version

Command 'dvc' not found, but can be installed with:

snap install dvc
```

It might be better to ask the user to install via `snap install --classic dvc` at first. 

Also
```
wget https://code.dvc.org/get-started/code.zip
unzip code.zip
rm code.zip
tree
git status -s
git add params.yaml src/
git status -s
```

~~commands should have `cd src` somewhere before `git status` and `rm code.zip`~~

Excuse me, I thought `code.zip` contains a git repository. That seems not the case. There is only a `.gitconfig` file at `~`. Where should the git repository be?

```sh
~$ ll -aR | grep .git
-rw-r--r--  1 root root   53 Feb 12 10:50 .gitconfig
```

---

The same problem seems to occur in https://katacoda.com/dvc/courses/get-started/pipelines 

I was trying to test #1586 but the whole scenario seems broken now.

**Comment by iesahin** [comment]
From the looks of https://github.com/iterative/katacoda-scenarios/blob/master/get-started/stages/assets/prepare.sh I think this issue is related with #2183 . `pip3 install dvc` line is probably failing and the waiting line `until ...` never ends.

**Comment by jorgeorpinel** [comment]
Hmmm interesting. Yeah we need to totally re-do them. Closing in favor of #1400. Thanks

**Comment by jorgeorpinel** [comment]
OK I guess since this is labeled bug and has a specific priority we can leave it for now.

**Comment by iesahin** [comment]
I think we can just close this. @jorgeorpinel It's a bug but can't be solved without more extensive changes described in #1400

**Comment by jorgeorpinel** [comment]
AGreed, should by fixed along with #1400. Ok thanks, closing.
